languages/wp-job-manager.pot

```diff
@@ -2770,10 +2769,6 @@ msgstr ""
 msgid "%1$s submitted successfully."
 msgstr ""
 
-#: tests/php/tests/includes/helper/test_class.wp-job-manager-helper.php:122
-msgid "Requires Attention"
-msgstr ""
-
```

Comment:
I'm not sure why this got removed if the file wasn't edited. :thinking:

Reply:
I think it is safe to ignore this, tbh. I tried running it locally and it made some even weirder changes to do this. :thinking: